staging/src/k8s.io/apiserver/pkg/registry/generic/registry/storage_factory.go

```diff
@@ -59,8 +59,12 @@ func StorageWithCacher(capacity int) generic.StorageDecorator {
 
 		// TODO: we would change this later to make storage always have cacher and hide low level KV layer inside.
 		// Currently it has two layers of same storage interface -- cacher and low level kv.
+		cacheCapacity := 0
```

Comment:
The `CacheCapacity` is ignored underneath anyway, so that doesn't really  matter.

We only care about capacity being nil/non-nil for the check above to decide whether we should use cacher or not.
This one actually doesn't matter, so let me remove it.